app/views/docs/production.phtml

```diff
@@ -13,38 +13,48 @@
 
 <h2><a href="/docs/production#limitConsoleAccess" id="limitConsoleAccess">Limit Console Access</a></h2>
 
-<p>By default, anyone can signup for your Appwrite server, create projects, and use your computing power. While this is great for testing around or running your Appwrite service in a network isolated environment, it is highly not recommended for public production use.</p>
```

Comment:
Is this and the paragraph below it still true?

Reply:
Update looks good